include/config/config_game.h

```diff
@@ -1,4 +1,19 @@
 #ifndef CONFIG_GAME_H
 #define CONFIG_GAME_H
 
+// Configure text speed (1 is vanilla)
+#define TEXT_SPEED 2
+
+// Rupee counter symbol changes color with wallet upgrades like in MM
+#define WALLET_COLORS
+
+// Wearing bunnyhood has the same functionality as in MM
+#define MM_BUNNYHOOD
+
+// Configure block push speed multiplier
+#define BLOCK_PUSH_SPEED 3
+
+// Replaces the low health beep sound with link blinking red
+#define NO_LOW_HEALTH_BEEP
```

Comment:
imo negative worded defines can get confusing, especially because you're defining something when it's getting disabled. I'd recommend flipping it like `ENABLE_LOW_HEALTH_BEEP`, and swapping the usage ofc